### ISSUE TITLE
Instruction text size

### ISSUE THREAD

**Issue by nole87** [mapbox/mapbox-navigation-android#3151]
**Mapbox Navigation SDK version:**
Newest one

Dear,

One issue, look attachment, how to change text size of instructions, not top one, only below top one.

Best,
Milan

![unnamed-4](https://user-images.githubusercontent.com/4254736/84301686-7fec0c00-ab54-11ea-8f98-5cd04ddcb207.jpg)

**Comment by Guardiola31337** [comment]
@JunDai Could you check if this is something we currently support in UI SDK `1.0`? If not, @asinghal22 is this something we should prioritize / work on?

cc @abhishek1508

**Comment by JunDai** [comment]
no, we don't support customizing text size.

**Comment by nole87** [comment]
In my opinion in top blue part text size looks great but below it's a big in my opinion it should be smaller. If it's possible please add it in new release of SDK ...

**Comment by P-Zenker** [comment]
Yeah, it would be great if we can adjust die size of the elements because sometimes not the whole text is displayed: 
![2020-07-07 17 40 11](https://user-images.githubusercontent.com/23097192/86806495-35897c80-c079-11ea-8784-d97d8532b12e.jpg)

Especially if we want to display out custom text in our custom navigation we would like to adjust the size. At the moment I adjust die size of stepPrimaryText / stepSecondaryText in instructionLayoutText programmatically by 
`requireView().findViewById<TextView>(R.id.stepPrimaryText)?.textSize = 24.0f`

Maybe the layout should reduce the text size by itself and if the size fall below of a certain threshold the three dots are included.

**Comment by JunDai** [comment]
@nole87 / @P-Zenker : thanks for your comments. I understand your concern and requirement. We will evaluate it and prioritize it accordingly.

**Comment by Guardiola31337** [comment]
cc'ing Design folks @d-prukop @avi-c @noracalabrese @map-angela and @asinghal22 for visibility

**Comment by nole87** [comment]
R.id.stepPrimaryText is for top text what id should I use for text steps below top text, that's what I must change so it looks great ... I try to find but for now nothing ...

**Comment by noracalabrese** [comment]
Thanks @Guardiola31337 
- I think the size could definitely be adjusted. And possibly the second line can be smaller than the first.
- If dynamic size is possible at some point, it would be good to still set a maximum and a minimum, so then we ensure that is readable.
Let me know if needed, we can create some mockups.
